pkg/k8shandler/exec.go

```diff
@@ -0,0 +1,81 @@
+package k8shandler
```

Comment:
@ewolinetz where do you put util code in CLO? I'd be happier if this file was somewhere else

Reply:
`pkg/utils/utils.go`